lms/views/api/exceptions.py

```diff
@@ -188,7 +186,7 @@ def api_error(self):
         if hasattr(self.context, "error_code"):
             return self.error_response(
                 error_code=self.context.error_code,
-                details=getattr(self.context, "extra_details", None),
+                details=getattr(self.context, "details", None),
```

Comment:
`details=details`, rather than `details=extra_details` (this is part of the erroneous renaming of `details` to `extra_details` mentioned above)